### ISSUE TITLE
NTR: Renal biopsy: vasculitis (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9413]
New term request Renal biopsy: vasculitis (segal_180706124754): Kidney  UMLS: 0151278

**Comment by pnrobinson** [comment]
@MickeySegal
@avizrosenberg @yongqunh

Can you give us a hand with this term request?

What about the definition

Vasculitis (inflammation of blood vessels) in the kidney. Renal vasculitis can be observed with numerous disorders and presents clinically as rapidly progressive glomerulonephritis (RPGN) which is characterised by acute kidney injury (AKI), haematuria and proteinuria. Histological examination of the kidney reveals severe inflammation in the form of crescent formation, glomerular necrosis and vasculitis of small and medium sized vessels within the kidney. 
PMID:31927782

https://pubmed.ncbi.nlm.nih.gov/31927782

**Comment by pnrobinson** [comment]
@MickeySegal

adding
Adding new term: 
- ID: HP:6000076
- label: Renal vasculitis
- parents: [Abnormal renal vascular morphology (HP:0033835)](https://hpo.jax.org/app/browse/term/HP:0033835)
- pmid: PMID:31927782
- def: Vasculitis (inflammation of blood vessels) in the kidney. Renal vasculitis can be observed with numerous disorders and presents clinically as rapidly progressive glomerulonephritis (RPGN) which is characterised by acute kidney injury (AKI), haematuria and proteinuria. Histological examination of the kidney reveals severe inflammation in the form of crescent formation, glomerular necrosis and vasculitis of small and medium sized vessels within the kidney.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9413
- synonyms:
